app/models/big_query_client.rb

```diff
@@ -19,4 +19,14 @@ def initialize
     self.client = Google::Cloud::Bigquery.new
   end
 
+  # clears the entire BQ table.  This is only intended for utility use in tests/development
+  # it will no-op in production
+  def self.clear_bq_table
+    if Rails.env.production?
+      return
+    end
+    client = BigQueryClient.new.client
+    query = "DELETE FROM #{CellMetadatum::BIGQUERY_TABLE} WHERE 1 = 1"
```

Comment:
Scary, even with the safeguards on line 25 and elsewhere.

Do we have a backup strategy for BigQuery?

Reply:
Since BQ isn't the ultimate source of truth, it really doesn't need a backup.  The 'backup' is just reingesting the metadata files.  Currently, reingesting all metadata files wouldn't take much more than an hour